action-allowedlist/approved.json

```diff
@@ -41,9 +41,9 @@
         "tag": "v4.1.0"
     },
     {
-      "actionLink": "actions/checkout",
-      "actionVersion": "b4ffde65f46336ab88eb53be808477a3936bae11",
-      "tag": "v4.1.1"
+        "actionLink": "actions/checkout",
+        "actionVersion": "b4ffde65f46336ab88eb53be808477a3936bae11",
+        "tag": "v4.1.1"
```

Comment:
Your editor appears to have changed all of the two-space "tabs" to four-spaces. Our general preference is for two spaces, which is what is set in https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-Actions/blob/main/.editorconfig

Which editor did you use on this file? Can you go back and reset to two spaces? That way we don't have the same churn when someone else edits and sends it back to two.

Reply:
Fixed